modules/apps/cookies/cookies-banner-web/src/main/java/com/liferay/cookies/banner/web/internal/display/context/BaseCookiesBannerDisplayContext.java

```diff
@@ -47,28 +49,43 @@ public BaseCookiesBannerDisplayContext(
 		cookiesConsentConfiguration = _getCookiesConfiguration(renderRequest);
 	}
 
-	public List<ConsentCookieType> getOptionalConsentCookieTypes(long groupId)
+	public List<ConsentCookieType> getOptionalConsentCookieTypes()
 		throws Exception {
 
 		if (_optionalConsentCookieTypes != null) {
 			return _optionalConsentCookieTypes;
 		}
 
-		_optionalConsentCookieTypes =
```

Comment:
@dsanz, with this change we can remove all methods in CookiesManagerUtil right? I noticed that we only use now add and get cookies

Reply:
Probably yes, I just took logic from the corresponding implementations from CookieManagerImpl to break the dependency with the `groupId` when preparing the context data. So the equivalent `getOptionalConsentCookieTypes()` and `getRequiredConsentCookieTypes()` in CookieManager might become unused